tabs/setup.js

```diff
@@ -34,6 +34,22 @@ TABS.setup.initialize = function (callback) {
 
     MSP.send_message(MSP_codes.MSP_ACC_TRIM, false, false, load_status);
 
+    //load current pid profile
+    function get_pid_controller() {
```

Comment:
get_pid_controller does not belong in the setup tab, it's used by the tuning tab to know if the pid controller can be changed. no other tab needs this functionality.

code to determine the current gui profile does not belong in the setup tab.